### PR TITLE
Fix the dependency of atorch.

### PR DESCRIPTION
**Pull request by youxingling**
### What changes were proposed in this pull request?

Fix the dependency of atorch.

### Why are the changes needed?

When I executed `pip install -r requirements.txt` before pretraining Llama2. I received an error message:

ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
atorch 0.1.5 requires fsspec==2023.1.0, but you have fsspec 2023.10.0 which is incompatible.

### Does this PR introduce any user-facing change?

NO.

### How was this patch tested?

UT.

### CODE REVIEW

File: atorch/examples/auto_accelerate/README.md [hunk 1]
```diff
@@ -12,32 +12,34 @@
 ## Usage
 
 ```
-train.py [-h] --model_type MODEL_TYPE [--datasize DATASIZE] [--epoch EPOCH] [--hiddien_size HIDDIEN_SIZE] [--head_num HEAD_NUM] [--layer_num LAYER_NUM] [--seq_length SEQ_LENGTH] [--batchsize BATCHSIZE] [--distributed] [--user_created_dataloader] [--load_strategy] [--optim_grouped_params] [--log_interval LOG_INTERVAL] [--use_fsdp] [--use_amp] [--use_checkpointing] [--use_module_replace]
+train.py [-h] --model_type MODEL_TYPE [--datasize DATASIZE] [--epoch EPOCH] [--hidden_size hidden_size] [--head_num HEAD_NUM] [--layer_num LAYER_NUM] [--seq_length SEQ_LENGTH] [--batchsize BATCHSIZE] [--distributed] [--user_created_dataloader] [--load_strategy] [--optim_grouped_params] [--log_interval LOG_INTERVAL] [--use_fsdp] [--use_amp] [--use_checkpointing] [--use_module_replace]
```
Comment (skydoorkai) [inline atorch/examples/auto_accelerate/README.md:15]: Keep upper letter for hidden_size to be consistent with other parameters.

**Comment by skydoorkai** [comment]
atorch 0.1.5 requires fsspec==2023.1.0, but you have fsspec 2023.10.0 which is incompatible.

Where does  fsspec 2023.10.0  come from? Installed by other package?

**Comment by youxingling** [comment]
fsspec 2023.10.0 comes from `atorch/examples/llama2/requirements.txt` and details are as follows :

```bash
Collecting fsspec<=2023.10.0,>=2023.1.0 (from fsspec[http]<=2023.10.0,>=2023.1.0->datasets>=2.14.6->-r requirements.txt (line 1))
  Downloading fsspec-2023.10.0-py3-none-any.whl.metadata (6.8 kB)
```

File: atorch/examples/auto_accelerate/README.md (hunk 1 above)
Comment (youxingling) [reply atorch/examples/auto_accelerate/README.md]: got it

**Review by skydoorkai — APPROVED** [review]
lgtm

**Pull request merged**